Source/DataSources/BillboardVisualizer.js

```diff
@@ -50,6 +50,7 @@ define([
     var EntityData = function(entity) {
         this.entity = entity;
         this.billboard = undefined;
+        this.textureValue = undefined;
```

Comment:
Most of the time this is string and `BillboardCollection` actually keeps a reference to it already.  The only thing BillboardCollection doesn't keep a reference to is the canvas objects, which we will now do here for the sake of comparing it against the original value.  However, in order for the object to be here, it has to exist somewhere else as well, so we are not really adding any extra memory into the system at all.